content/tutorials/get-started/get-started-with-rippleapi-for-javascript.md

```diff
@@ -208,15 +228,18 @@ If you are the administrator of the `rippled` server, you can [manually request
 
 See [Reliable Transaction Submission](reliable-transaction-submission.html) for a more thorough explanation.
 
-# RippleAPI in Web Browsers
+
+
+## Use RippleAPI in a Web Browser
 
 RippleAPI can also be used in a web browser if you compile a browser-compatible version and include [lodash](https://www.npmjs.com/package/lodash) as a dependency before the RippleAPI script.
 
-## Build Instructions
+
+### Build a Browser-Compatible Version of RippleAPI
 
 To use RippleAPI in a browser, you need to build a browser-compatible version. The following process compiles RippleAPI into a single JavaScript file you can include in a webpage.
 
-#### 1. Download a copy of the RippleAPI git repository.
+#### 1. Download a copy of the RippleAPI git repository
```

Comment:
If we split this into its own page, I'd actually like to see this raised to h3-level so it appears in the sidebar (like the steps in our other tutorials)

Reply:
I hear you on splitting the tutorial up into separate pages. I'm going to leave it as-is for this ticket and will open a new ticket for the actual separation.